### PR TITLE
feat: method to get xpubkey using PIN to decrypt data

### PR DESCRIPTION
**Pull request by pedroferreira1**

### CODE REVIEW

File: src/wallet.js [hunk 1]
```diff
@@ -2035,6 +2035,21 @@ const wallet = {
...
   },
+
+  /**
+   * Get xpubkey from storage xpriv (assumes PIN is correct)
+   *
+   * @param {String} pin User PIN used to encrypt xpriv on storage
+   *
+   * @return {String} Wallt xpubkey
+   */
+  getXPubFromPin(pin) {
```
Comment (msbrogli) [inline src/wallet.js:2046]: I'd name it `getXPubKeyFromXPrivKey(int)`. What do you think?

File: src/wallet.js [hunk 2]
```diff
@@ -2035,6 +2035,21 @@ const wallet = {
     const key = hdpubkey.derive(index);
     return key.publicKey.toBuffer();
   },
+
+  /**
+   * Get xpubkey from storage xpriv (assumes PIN is correct)
+   *
+   * @param {String} pin User PIN used to encrypt xpriv on storage
+   *
+   * @return {String} Wallt xpubkey
```
Comment (obiyankenobi) [inline src/wallet.js:2044]: typo `Wallt`

File: src/wallet.js (hunk 1 above)
Comment (pedroferreira1) [reply src/wallet.js:2046]: Changed!

File: src/wallet.js (hunk 2 above)
Comment (pedroferreira1) [reply src/wallet.js:2044]: Thanks

**Pull request merged**